commit 45d4dfed1ded628abcbb9f1c62dfb90095414f0d
Author: Sage Weil <sweil@redhat.com>
Date:   Tue Jun 5 13:57:09 2018 -0500

    mgr: load persistent device metadata on mgr stat
    
    Prune DeviceState when there is no persistent metadata and no daemon
    references.  Load persistent metadata from config-key.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/PyModuleRegistry.cc b/src/mgr/PyModuleRegistry.cc
index 3ceb4d0e939..af1fdebbb90 100644
--- a/src/mgr/PyModuleRegistry.cc
+++ b/src/mgr/PyModuleRegistry.cc
@@ -412,7 +412,7 @@ void PyModuleRegistry::upgrade_config(
     dout(1) << "Upgrading module configuration for Mimic" << dendl;
     // Upgrade luminous->mimic: migrate config-key configuration
     // into main configuration store
-    for(auto &i : old_config) {
+    for (auto &i : old_config) {
       auto last_slash = i.first.rfind('/');
       const std::string module_name = i.first.substr(4, i.first.substr(4).find('/'));
       const std::string key = i.first.substr(last_slash + 1);
